packages/help-center/src/hooks/use-content-filter.ts

```diff
@@ -61,6 +61,23 @@ export const useContentFilter = ( node: HTMLDivElement | null ) => {
 					};
 				},
 			},
+			{
+				pattern: 'iframe[data-wpcom-embed-url*="videopress.com"]',
+				action: ( element ) => {
```

Comment:
```suggestion
				action: ( element: HTMLElement ) => {
```

Just to maintain consistent typing